tutorial-5/src/eth_prices/store.rs

```diff
@@ -306,7 +314,13 @@ mod tests {
 
     #[test_context(TestDb)]
     #[tokio::test]
+    #[serial]
```

Comment:
serial enables test cases execute in order in this way to avoid async test cases modify the same database table caused test result not as expected value.